docs/arxiv/empirical/experimentation.tex

```diff
@@ -1,2 +1,12 @@
 \section{Experimentation}
 \subsection{Experiment 1: Learning Q functions on components of the Actor network.}
+
+\textbf{Motivation:}
+Our first batch of experiments aimed to establish empirically that the Q-functions of each layer, denoted $Q_1...Q_L$ are similar to the Q-function of the overall network, denoted $Q_{\mu}$. To that end, we extended the actor-critic methodology used in Lillicrap et al (2016) as follows. In addition to training $Q^\mu$, to estimate the Q functions of the actor network $\mu$, sub-critic networks $Q^n$ were initialized for each individual layer. We then compare the $Q$ values estimated by the subcritics to the $Q$ estimation provided by the main critic, $Q^\mu$. \\
+\newline
+The inputs used in the calculation of the Q-functions for each layer follow the conceptual framework laid out in Section 2.2. The state and action parameters are therefore voltages represented by the outputs of $\epsilon : \scripts \to \scriptv$ and $\delta : \scriptv \to \scripta$ respectively, where the changes in voltage are described by a voltage transition function $K : \scriptv \to \scriptv$. \\
```

Comment:
It may be easier to say something like: According to the framework the state and action input to each $Q^n$ is the activations of every anterior neuron to $n$ and the output of neuron $n$ respectively.

You can fluff this up a bit.